### ISSUE TITLE
Ordering and display options in tool panel

### ISSUE THREAD

**Issue by mvdbeek** [galaxyproject/galaxy#12251]
Might be nice to have a few options, like displaying the tool id instead of the name, alphabetic ordering, flat list instead of sections, etc.

**Comment by natefoo** [comment]
Labels might be a bit tricky, you probably want things to stay under their relevant labels?

**Comment by dannon** [comment]
I think this would fold nicely into the tool suites work, I'll see if it makes sense to integrate it there.

**Comment by mvdbeek** [comment]
> Labels might be a bit tricky, you probably want things to stay under their relevant labels?

Do you mean the section labels ?
I think they can be in the way for tools that could be in multiple sections. It shouldn't be the default view, but I think that's a useful view if you know you want to run seqtk, and so you know it should start with `s`.

**Comment by dannon** [comment]
@mvdbeek Yep, the way this was set up "Tools ordered Alphabetically" could simply be one of the options in the dropdown -- great idea.  I will try to dig it up.
